target/linux/ramips/dts/mt7628an_tplink_tl-wr902ac-v4.dts

```diff
@@ -0,0 +1,112 @@
+#include "mt7628an_tplink_8m.dtsi"
```

Comment:
Please add the GPL license here. `// SPDX-License-Identifier: GPL-2.0-or-later OR MIT`

Reply:
Well, I have one doubt. This is modified copy of mt7628an_tplink_tl-wr902ac-v3.dts and in that file there were no copyright information. So maybe I had no right to use it here :( ?

Reply:
It's okay. And inserting a new line after license would be better.
https://github.com/openwrt/openwrt/blob/36b1dd75fd9da1ea13f1ce1ee679c6b3c9c402cc/COPYING#L1-L12